src/AWS.Deploy.Orchestration/ServiceHandlers/AWSElasticBeanstalkHandler.cs

```diff
@@ -37,6 +82,39 @@ public AWSElasticBeanstalkHandler(IAWSClientFactory awsClientFactory, IOrchestra
             _optionSettingHandler = optionSettingHandler;
         }
 
+        /// <summary>
+        /// Elastic Beanstalk Windows deployments require a manifest file to be included with the application to be deployed.
+        /// This function creates the windows manifest file and adds it to the zip file that will be deployed to Beanstalk.
+        /// </summary>
+        public void SetupWindowsDeploymentManifest(Recommendation recommendation, string dotnetZipFilePath)
```

Comment:
The way the legacy wizard works is it checks to see if the project already contains a manifest file and if applies the IIS app path and web site setting without losing any other settings in the file. Users can actually make a lot more IIS setting changes then just app path and web settings. We need to make sure we don't lose that functionality when migrating to this experience. Here is the old code for creating the manifest.

https://github.com/aws/aws-extensions-for-dotnet-cli/blob/0d788f1015acbe836c4a4d8961f50e07c7924076/src/Amazon.ElasticBeanstalk.Tools/EBUtilities.cs#L16

Reply:
Fixed